pkg/apis/certmanager/v1alpha2/types_issuer.go

```diff
@@ -147,7 +147,9 @@ type VenafiCloud struct {
 	APITokenSecretRef cmmeta.SecretKeySelector `json:"apiTokenSecretRef"`
 }
 
-type SelfSignedIssuer struct{}
+type SelfSignedIssuer struct {
+	CRLDistributionPoints []string `json:"crlDistributionPoints,omitempty"`
```

Comment:
Please add a comment on this field explaining:

* What it does
* What values are valid
* What the default behaviour is if not specified

You should also have a `// +optional` line (similar to some other fields in this file) as the field is optional 😄

Reply:
This text is displayed to users through both our docs and `kubectl explain` so it really helps to provide clearly information about fields and how they should be used 😄

Reply:
Ah, we can also include `// +kubebuilder:validation:UniqueItems` here too to avoid users specifying duplicates.. not sure if it's strictly required, but it's easier to relax validation in future than it is to enforce additional validation 😄